### DISCUSSION TITLE
autoComplete prevents submitting the form by enter key

### DISCUSSION THREAD

**Discussion by Solution97** [TarekRaafat/autoComplete.js#261] [Q&A] [answered]
Hey guys,

I noticed that when I have a form without a button, that i can't submit if the autocomplete suggestions are visible.
Is there an option to enable "submitting by enter key" or do I need to implement it by my self ?

Have a nice day!

**Comment by folknor** [comment, marked as answer]
https://github.com/TarekRaafat/autoComplete.js/blob/master/src/autoComplete.js#L46

```js
new autoComplete({
	submit: true,
})
```

**Comment by TarekRaafat** [comment]
Hello @Solution97,

Adding to @folknor, here is the [docs](https://tarekraafat.github.io/autoComplete.js/#/configuration?id=submit-optional) section of this specific point.

Thanks @folknor, and have a nice day guys!
